extensions/UserProfile/template/en/default/pages/user_profile.html.tmpl

```diff
@@ -108,6 +108,13 @@
       [% END %]
     </td>
   </tr>
+[% ELSE %]
+  <tr>
+    <th>Get Permissions</th>
+    <td>
```

Comment:
pretty sure there is a colspan="2" needed here, were you able to test this? need help with the vagrant vm?